Tools/WinMLRunner/src/Run.cpp

```diff
@@ -178,12 +173,7 @@ HRESULT EvaluateModel(
         return hr.code();
     }
 
-    if (!args.Silent())
-    {
-        std::cout << "[SUCCESS]" << std::endl;
-    }
-
-    if (!isGarbageData && !args.Silent())
+    if (capturePerf && !isGarbageData)
```

Comment:
If we want to capture performance and we included our own input and then iterated 100 times, processing and printing out the results would be expensive